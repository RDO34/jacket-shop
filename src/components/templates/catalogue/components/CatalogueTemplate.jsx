import React from 'react';
import { Hero } from '../../../atoms';

export const CatalogueTemplate = () => (
  <>
    <Hero src="placeholder" />
  </>
);
