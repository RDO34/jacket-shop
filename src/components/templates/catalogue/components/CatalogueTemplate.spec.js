import React from 'react';
import { render } from 'react-testing-library';

import { CatalogueTemplate } from "./CatalogueTemplate";

describe('catalogue template', () => {
  it('renders the hero component', () => {
    const { getByTestId } = render(<CatalogueTemplate />);
    expect(getByTestId('hero-atom')).toBeTruthy();
  });

  it('matches the snapshot', () => {
    const { container } = render(<CatalogueTemplate />);
    expect(container.firstChild).toMatchSnapshot();
  })
})
