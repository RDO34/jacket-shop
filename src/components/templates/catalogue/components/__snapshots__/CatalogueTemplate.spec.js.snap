// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`catalogue template matches the snapshot 1`] = `
<div
  class="css-h3xb4e"
  data-testid="hero-atom"
  src="placeholder"
/>
`;
